src/refmt_impl.ml

```diff
@@ -167,12 +191,14 @@ let () =
         )
         | Some "binary"
         | None -> fun (ast, comments) -> (
-          output_string stdout Config.ast_intf_magic_number;
-          output_value  stdout filename;
-          output_value  stdout ast
+            let (module V) = !ocaml_version in
```

Comment:
@let-def How do we automatically know which version of the ocaml compiler to serialize to? Is it automatically handled by the migrate-parsetree package?